### PR TITLE
material file for signage.mt

### PR DESCRIPTION
**Pull request by Simarilius-uk**
Its the neon signs used in a lot of the items in base\environment\decoration\advertising\signage such as neon_kiroshi_optics_logo or sex_shop_neon_sign_h.
Not perfect yet but a lot better than grey.

### CODE REVIEW

File: i_scene_cp77_gltf/material_types/signages.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            dCol = CreateShaderNodeRGB(CurMat, Data["ColorThree"], -950, 250, "ColorThree")    
+
+        if "ColorFour" in Data:
+            dCol = CreateShaderNodeRGB(CurMat, Data["ColorFour"], -1000, 250, "ColorFour")    
+                    
+        if "ColorFive" in Data:
+            dCol = CreateShaderNodeRGB(CurMat, Data["ColorFive"], -1050, 250, "ColorFive")  
+        
+        if "ColorSix" in Data:
+            dCol = CreateShaderNodeRGB(CurMat, Data["ColorSix"], -1100, 250, "ColorSix")
```
Comment (dragonzkiller) [inline i_scene_cp77_gltf/material_types/signages.py:68]: Are these not connected to anything yet?

**Review by dragonzkiller — CHANGES_REQUESTED** [review]

File: i_scene_cp77_gltf/material_types/signages.py (hunk 1 above)
Comment (Simarilius-uk) [reply i_scene_cp77_gltf/material_types/signages.py:68]: No, I haven't found any that use it to work out what it was doing but the data points were there in the mt, figured this way if you do hit one it at least will show you the color.

File: i_scene_cp77_gltf/material_types/signages.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            emTexNode = CreateShaderNodeTexImage(CurMat, self.BasePath + Data["MainTexture"], -700, -250,'MainTexture',self.image_format)
+            CurMat.links.new(emTexNode.outputs[0],mulNode.inputs[2])
+            CurMat.links.new(emTexNode.outputs[1],alphaNode.inputs[0])
+
+
+        CurMat.links.new(alphaNode.outputs[0], CurMat.nodes['Principled BSDF'].inputs['Alpha'])
+        CurMat.links.new(mulNode.outputs[0], CurMat.nodes['Principled BSDF'].inputs['Emission'])
+        
+        if "EmissiveEV" in Data:
+            CurMat.nodes['Principled BSDF'].inputs['Emission Strength'].default_value =  Data["EmissiveEV"]*10
```
[the remark this answers is not in the document]
Comment (Simarilius-uk) [reply i_scene_cp77_gltf/material_types/signages.py:44]: honestly struggling to remember as I wrote it a month ago, think it was mainly from comparing in game to blender, the in game neons are basically white with the color visible in the glow, without the 10 they were just vaguely glowing colored lines.

[the remark this answers is not in the document]
Comment (dragonzkiller) [reply i_scene_cp77_gltf/material_types/signages.py:44]: A lot of that look can come from the environment (i.e. if the air in a room is dusty). I'm not sure if we should include it in the actual plugin

File: i_scene_cp77_gltf/material_types/signages.py (hunk 1 above)
Comment (dragonzkiller) [reply i_scene_cp77_gltf/material_types/signages.py:68]: We should probably investigate a few more that use that template before just letting them sit there

**Pull request closed without merging**